core/src/main/scala/io/finch/internal/currentTime.scala

```diff
@@ -1,12 +1,12 @@
 package io.finch.internal
 
+import java.time.{Instant, ZoneId}
 import java.time.format.DateTimeFormatter
-import java.time.Instant
-import java.time.ZoneOffset
+
 
 object currentTime {
   private[this] val formatter: DateTimeFormatter =
-    DateTimeFormatter.RFC_1123_DATE_TIME.withZone(ZoneOffset.UTC)
+    DateTimeFormatter.ofPattern("EEE, dd MMM yyyy HH:mm:ss zzz").withZone(ZoneId.of("GMT"))
```

Comment:
I think [Play](https://github.com/playframework/playframework/blob/38abd1ca6d17237950c82b1483057c5c39929cb4/framework/src/play/src/main/scala/play/api/http/package.scala#L20) does the same thing as Finatra. [Akka Http](https://github.com/akka/akka-http/blob/5932237a86a432d623fafb1e84eeeff56d7485fe/akka-http-core/src/main/java/akka/http/javadsl/model/DateTime.java#L90) uses its own mechanism, but the comment there seems to indicate its always "GMT" and nothing needs a Locale. I'm kinda tending toward `Locale.English`